docs/external/faq.md

```diff
@@ -36,6 +36,8 @@ Edge release: A monthly release with new features and bug fixes.
 
 Stable release: A quarterly release with reliable/stable updates.
 
+***Note: From 0.19 release of vDVS, the plugin has to have a ESX service thats at least 0.19 or later. This applies for plugins on Linux platform.***
```

Comment:
Let's not add here and move out as a separate FAQ entry for any user to search error message to get quick answer.

Reply:
Done